blackfriday-bingo.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Komplett.JsonClient" Version="2.3.0" />
```

Comment:
You can't use Komplett.JsonClient, because this project is not built within the firewall of Komplett 😸